### PR TITLE
restore compatibility with legacy version v5.32

### PR DESCRIPTION
**Pull request by rjbs**

### CODE REVIEW

File: lib/Synergy/Reactor/Linear.pm [hunk 1]
```diff
@@ -748,12 +747,19 @@ command agenda => {
       } else {
         my ($assignee_id, $team_id);
 
-        try {
+        my $error_reply;
```
Comment (wolfsage) [inline lib/Synergy/Reactor/Linear.pm:750]: nit: not used

File: lib/Synergy/Reactor/Linear.pm [hunk 2]
```diff
@@ -871,12 +877,16 @@ async sub _handle_creation_event ($self, $event, $arg = {}) {
     $text =~ s/\A> >/>>/;
 
     # XXX: I do not like our current error-returning scheme. -- rjbs, 2021-12-10
-    my $plan;
-    try {
+    my ($plan, $error_reply);
```
Comment (wolfsage) [inline lib/Synergy/Reactor/Linear.pm:880]: nit: not used (error_reply)

File: lib/Synergy/Reactor/RememberTheMilk.pm [hunk 3]
```diff
@@ -217,11 +226,16 @@ command milkauth => {
       …and then tell me `milkauth complete`
       END
 
-      return await $event->private_reply($text, { slack => $text });
-    } catch ($fail) {
-      $Logger->log([ "failed to make start auth: %s", $fail ]);
-      return await $event->reply("Sorry, something went wrong!");
+      return $event->private_reply($text, { slack => $text });
```
Comment (wolfsage) [inline lib/Synergy/Reactor/RememberTheMilk.pm:229]: not a return await which seems weird

File: lib/Synergy/Reactor/Zendesk.pm [hunk 4]
```diff
@@ -249,18 +254,23 @@ async sub ticket_report ($self, $who, $arg = {}) {
   }
 
   my $res;
-  try {
+  my $to_return;
```
Comment (wolfsage) [inline lib/Synergy/Reactor/Zendesk.pm:257]: nit: unused

File: lib/Synergy/Reactor/RememberTheMilk.pm (hunk 3 above)
Comment (rjbs) [reply lib/Synergy/Reactor/RememberTheMilk.pm:229]: This returns it from the eval, after which we `return await` it.  We need to return out of the eval.

Comment (wolfsage) [reply lib/Synergy/Reactor/RememberTheMilk.pm:229]: Ah! Okay

**Pull request merged**